content/en/blog/_posts/2023-09-15-Bootstrap-an-Air-Gapped-Cluster-with-Kubeadm/index.md

```diff
@@ -0,0 +1,665 @@
+```
+Now pivot over to the air gap VM for the rest of the installation procedure.
+
+Switch into ~/tmp:
+```
+cd ~/tmp
+```
+Extract and move the helm binary:
+
+```
+tar -zxvf helm-v3.0.0-linux-amd64.tar.gz
+mv linux-amd64/helm /usr/local/bin/helm
+```
+
+**Load the Podinfo container image into the local Docker registry:**
+
+```bash
+docker load -i podinfo_podinfo-6.4.0.tar
+docker tag podinfo/podinfo:6.4.0 localhost:5000/podinfo/podinfo:6.4.0
+docker push localhost:5000/podinfo/podinfo:6.4.0
+```
+
+**Ensure the proper KUBECONFIG is set and install the Podinfo helm chart:**
+
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+helm install podinfo ./podinfo-6.4.0.tgz --set image.repository=localhost:5000/podinfo/podinfo
+```
+
+Verify that the Podinfo application comes up:
+```bash
+kubectl get pods -n default
+```
+Or run k9s (a terminal user interface for Kubernetes):
+```bash
+k9s
+```
+## Zarf
+
+Zarf is an open-source tool that takes a declarative approach to software packaging and delivery, including air gap. This same podinfo application will be installed onto the air gap VM using Zarf in this section. The first step is to install [Zarf](https://zarf.dev/install/) on the host/laptop.
+
+Alternatively, a prebuilt binary can be downloaded onto the host/laptop from [GitHub](https://github.com/defenseunicorns/zarf/releases/) for various OS/CPU architectures.
+
+A binary is also needed across the air gap on the VM:
+```bash
+UARCH=$(uname -m)
+# Set the architecture variables if needed
+if [["$UARCH" == "arm64" || "$UARCH" == "aarch64"]]; then
+
+    ARCH="aarch64"
+    K8s_ARCH="arm64"
+
+else
+
+    ARCH="x86_64"
+    K8s_ARCH="amd64"
+
+fi
+
+export ZARF_VERSION=v0.28.3
+
+curl -LO "https://github.com/defenseunicorns/zarf/releases/download/${ZARF_VERSION}/zarf_${ZARF_VERSION}_Linux_${K8s_ARCH}"
+```
+Zarf needs to bootstrap itself into a Kubernetes cluster through the use of an init package. That also needs to be transported across the air gap so let's download it onto the host/laptop:
+```bash 
+curl -LO "https://github.com/defenseunicorns/zarf/releases/download/${ZARF_VERSION}/zarf-init-${K8s_ARCH}-${ZARF_VERSION}.tar.zst"
+```
+The way that Zarf is declarative is through the use of a zarf.yaml file. Here is the zarf.yaml file that will be used for this Podinfo installation. Write it to whatever directory you you have write access to on your host/laptop; your home directory is fine:
+```
+echo "kind: ZarfPackageConfig
+metadata:
+    name: podinfo
+    description: "Deploy helm chart for the podinfo application in K8s via zarf"
+components:
+    - name: podinfo
+        required: true
+        charts:
+            - name: podinfo
+              version: 6.4.0
+              namespace: podinfo-helm-namespace
+              releaseName: podinfo
+              url: https://stefanprodan.github.io/podinfo
+        images:
+        - ghcr.io/stefanprodan/podinfo:6.4.0" > zarf.yaml
+```
```

Comment:
This quoting looks wrong. Use single quotes for the outer ones **or** use a multiline YAML string for the description.

Reply:
Yup, whoops.  I'll do single quotes for the outer ones.